### PR TITLE
Fix Modal close button is too small #557

### PR DESCRIPTION
**Pull request by tikhsuP**
## Description
Made the required changes.

### CODE REVIEW

File: src/assets/sass/protocol/components/_modal.scss [hunk 1]
```diff
@@ -81,7 +81,7 @@ html.mzp-is-noscroll {
     .mzp-c-modal-button-close {
         @include image-replaced;
         background: transparent url('#{$image-path}/icons/close-white.svg') center center no-repeat;
-        @include background-size(16px 16px);
+        @include background-size(30px 30px);
```
Comment (stephaniehobson) [inline src/assets/sass/protocol/components/_modal.scss]: Thank you for the pull request. Could you please make this `20px 20px` to match the same size the icons on the menu and details components.

Comment (tikhsuP) [reply src/assets/sass/protocol/components/_modal.scss]: Yes, definitely.

Comment (tikhsuP) [reply src/assets/sass/protocol/components/_modal.scss]: @stephaniehobson 
The requested changes have been made, kindly review.
Thanks

**Comment by craigcook** [comment]
I rebased, updated the changelog, and merged this manually in https://github.com/mozilla/protocol/commit/ce77ad5b6a03d336330d45b3d1c8253323ad1bf3

Thanks for the pull request!

**Comment by tikhsuP** [comment]
@craigcook Thanks.

**Pull request closed without merging**